app/views/dishes_templates/show.html.erb

```diff
@@ -1,160 +1,76 @@
-<div class="page-header">
```

Comment:
These changes to HTML seem to make sense even though the diff is kind of meaningless because the original page bore no resemblance whatsoever to what we were shooting for.